tests/Feature/Http/Controllers/RatingControllerTest.php

```diff
@@ -35,12 +36,18 @@ public function setUp(): void
     public function index_displays_view()
     {
         $ratings = Rating::factory()->count(3)->create();
+        $users = User::factory(3)->has(Rating::factory()->count(3))->has(Session::factory())->create();
 
-        $response = $this->get(route('rating.index'));
+        $response = $this->withoutExceptionHandling()->get(route('rating.index', [
+            'name' => $users->first()->name,
+            'search_language' => $users->first()->sessions->first()->language,
+            'from_event' => $users->first()->ratings->first()->entry,
+            'search_category' => 'CREATIVE',
+            ]));
 
         $response->assertOk();
         $response->assertViewIs('rating.index');
-        $response->assertViewHas('ratings');
+        $response->assertViewHas('users');
```

Comment:
Something that i think is important to recognize about this test is that while it may _cover_ everything, it doesn't really test the functionality. That said I'm going to say this is ok for now though since we're going to need to modify this test later when we implement the front-end anyway